public/app/features/dashboard-scene/settings/variables/components/CustomVariableForm.tsx

```diff
@@ -71,39 +67,3 @@ export function CustomVariableForm({
     </>
   );
 }
-
-export function getCustomVariableOptions(variable: SceneVariable): OptionsPaneItemDescriptor[] {
```

Comment:
not sure why this was moved from here.  just trying to understand the structure.  CustomVariableForm is for the settings page, while CustomVariableEditor is for the edit pane?